### PR TITLE
FLUT-7154 - [Feature] UG documentation has been added to DataGrid for the sorting number style and checkbox shape enhancements.

### PR DESCRIPTION
**Pull request by Tamilarasan-Paranthaman**
## Description ##

* UG documentation has been added to DataGrid for the `sort order number and it's background color change` and `checkbox shape change` enhancements.

### CODE REVIEW

File: Flutter/datagrid/sorting.md [hunk 1]
```diff
@@ -347,6 +347,68 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows sort sequence numbers during multi-column sorting](images/sorting/flutter-datagrid-showsortnumbers.gif)
 
+## Change sort number and backgroud color.
+
+The color of the sort number and its background color can be customized by using `SfDataGridThemeData.sortOrderNumberColor` and `SfDataGridThemeData.sortOrderNumberBackgroundColor` respectively.
```
Comment (NeelakandanKannan) [inline Flutter/datagrid/sorting.md:352]: The color of the sort order number and its rounded background

File: Flutter/datagrid/sorting.md [hunk 2]
```diff
@@ -347,6 +347,68 @@ Widget build(BuildContext context) {
...
+
+The following code describes how to change sort number and background color by using [SfDataGridTheme](https://pub.dev/documentation/syncfusion_flutter_core/latest/theme/SfDataGridTheme-class.html).
```
Comment (NeelakandanKannan) [inline Flutter/datagrid/sorting.md:354]: This line is not required.

**Review by NeelakandanKannan — CHANGES_REQUESTED** [review]
Get back to me with the changes.

File: Flutter/datagrid/sorting.md (hunk 1 above)
Comment (Tamilarasan-Paranthaman) [reply Flutter/datagrid/sorting.md:352]: Modified

File: Flutter/datagrid/sorting.md (hunk 2 above)
Comment (Tamilarasan-Paranthaman) [reply Flutter/datagrid/sorting.md:354]: I have removed

**Pull request merged**